<Gene_Name SCN9A />

<GeneName_full sodium channel protein type 9 subunit alpha />

# What does the SCN9A gene do?

SCN9A (sodium channel protein type 9 subunit alpha) control [sodium channels](http://www.uniprot.org/citations/17145499) in neurons that are part of the autonomic (involuntary) nervous system.  The channels are controlled by voltage differences across membranes and help to [feel pain](http://www.uniprot.org/citations/17145499) and develop [inflammatory pain](http://www.uniprot.org/citations/17167479).  Numerous diseases are caused by [variants](https://www.ncbi.nlm.nih.gov/pubmed/23129781) in SCN9A, such as [congenital insensitivity or indifference to pain (CIP)](https://www.ncbi.nlm.nih.gov/pubmed/20635406), [primary erythromelalgia (PERYTHM)](https://www.ncbi.nlm.nih.gov/pubmed/14985375), [paroxysmal extreme pain disorder (PEPD)](https://www.ncbi.nlm.nih.gov/pubmed/17145499), [generalized epilepsy with febrile seizures](https://www.ncbi.nlm.nih.gov/pubmed/19763161), [fibromyalgia](https://www.ncbi.nlm.nih.gov/pubmed/29392201), and [CFS](https://www.ncbi.nlm.nih.gov/pubmed/29392201). 

This gene is located on chromosome 2. The protein it creates acts in your nervous system and brain.

<# TissueList brain D001921 #>

<TissueList D001921 />

<GeneAnalysis gene="SCN9A" interval="NC_000002.12:g.166195185_166375987">

# What are some common mutations of SCN9A?
 
There are six well-known variants in SCN9A: [T166298928G](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=6754031), [C984A (Tyr328Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6363/), [C829T (Arg277Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6362/), [C2986T (Arg996Cys)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6356/), [G2691A (Trp897Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6355/), and [G1376C (Ser459Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6353/).

<# T166298928G #>
  <Variant hgvs="NC_000002.12:g.166298928T>G" name="T166298928G"> 

    This variant is a change at a specific point in the SCN9A gene from thymine (T) to guanine (G) resulting in incorrect protein function. This substitution of a single nucleotide is known as a missense variant.
 
  </Variant>
<# C984A #>
  <Variant hgvs="NC_000002.12:g.166293354G>T" name="C984A"> 

    This variant is a change at a specific point in the SCN9A gene from cytosine (C) to adenine (A) resulting in incorrect protein function. This substitution of a single nucleotide is known as a missense variant.

  </Variant>
<# C829T #>
  <Variant hgvs="NC_000002.12:g.166303162G>A" name="C829T"> 

    This variant is a change at a specific point in the  gene from cytosine (C) to thymine (T) resulting in incorrect six function. This substitution of a single nucleotide is known as a missense variant.

  </Variant>
<# C2986T #>
  <Variant hgvs="NC_000002.12:g.166272731G>A" name="C2986T"> 

    This variant is a change at a specific point in the SCN9A gene from cytosine (C) to thymine (T) resulting in incorrect protein function. This substitution of a single nucleotide is known as a missense variant.

  </Variant>
<# G2691A #>
  <Variant hgvs="NC_000002.12:g.166277133C>T" name="G2691A"> 

    This variant is a change at a specific point in the SCN9A gene from guanine (G) to adenine (A) resulting in incorrect protein function. This substitution of a single nucleotide is known as a missense variant.

  </Variant>
<# G1376C #>
  <Variant hgvs="NC_000002.12:g.166286562G>C" name="G1376C"> 

    This variant is a change at a specific point in the SCN9A gene from cytosine (C) to guanine (G) resulting in incorrect protein function. This substitution of a single nucleotide is known as a missense variant.

  </Variant>

<# T166298928G #>
  <Genotype hgvs="NC_000002.12:g.[166298928T>G];[166298928=]" name="T166298928G"> 

    # What does this mean?
 
    People with this variant have one copy of the [T166298928G](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=6754031)

    # What is the effect of this variant?

    This variant is not associated with increased risk.

    # How common is this genotype in the general population?

    <piechart percentage=45.8 />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166298928T>G];[166298928T>G]" name="T166298928G"> 
 
    # What does this mean?

    People with this variant have two copies of the [T166298928G](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=6754031) variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    You are in the Moderate Loss of Function category. See below for more information.

    # How common is this genotype in the general population?

    <piechart percentage=24.1 />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166298928=];[166298928=]" name="T166298928G"> 
 
    # What does this mean?

    Your SCN9A gene has no variants. A normal gene is referred to as a "wild-type" gene.

    # What is the effect of this variant?

    This variant is not associated with increased risk.

    # How common is this genotype in the general population?

    <piechart percentage=30.2 />
  </Genotype>
<# C984A #>
  <Genotype hgvs="NC_000002.12:g.[166298928T>G];[166298928=]" name="C984A"> 

    # What does this mean?
 
    People with this variant have one copy of the [C984A (Tyr328Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6363/) variant. This substitution of a single nucleotide is known as a missense mutation.

    This variant is a change at a specific point in the  gene from  to  resulting in incorrect  function. This substitution of a single nucleotide is known as a missense variant.

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=0.01 />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166293354G>T];[166293354G>T]" name="C984A"> 
 
    # What does this mean?

    People with this variant have two copies of the [C984A (Tyr328Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6363/) variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    You are in the Severe Risk category. See below for more information.

    # How common is this genotype in the general population?

    <piechart percentage=0.01 />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166293354=];[166293354=]" name="C984A"> 
 
    # What does this mean?

    Your SCN9A gene has no variants. A normal gene is referred to as a "wild-type" gene.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=99.98 />
  </Genotype>
<# C829T #>
  <Genotype hgvs="NC_000002.12:g.[166303162G>A];[166303162=]" name="C829T"> 

    # What does this mean?
 
    People with this variant have one copy of the C829T variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=0.01 />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166303162G>A];[166303162G>A]" name="C829T"> 
 
    # What does this mean?

    People with this variant have two copies of the C829T variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    You are in the Severe Risk category. See below for more information.

    # How common is this genotype in the general population?

    <piechart percentage=0.01 />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166303162=];[166303162=]" name="C829T"> 
 
    # What does this mean?

    Your SCN9A gene has no variants. A normal gene is referred to as a "wild-type" gene.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=99.98 />
  </Genotype>
<# C2986T #>
  <Genotype hgvs="NC_000002.12:g.[166272731G>A];[166272731=]" name="C2986T"> 

    # What does this mean?
 
    People with this variant have one copy of the [C2986T (Arg996Cys)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6356/) variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=0.1 />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166272731G>A];[166272731G>A]" name="C2986T"> 
 
    # What does this mean?

    People with this variant have two copies of the [C2986T (Arg996Cys)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6356/) variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    You are in the Severe Risk category. See below for more information.

    # How common is this genotype in the general population?

    <piechart percentage=0.01 />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166272731=];[166272731=]" name="C2986T"> 
 
    # What does this mean?

    Your SCN9A gene has no variants. A normal gene is referred to as a "wild-type" gene.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=99.88 />
  </Genotype>
<# G2691A #>
  <Genotype hgvs="NC_000002.12:g.[166277133C>T];[166277133=]" name="G2691A"> 

    # What does this mean?
 
    People with this variant have one copy of the [G2691A (Trp897Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6355/) variant. This substitution of a single nucleotide is known as a missense mutation.

    This variant is a change at a specific point in the  gene from  to  resulting in incorrect NC_000002.12:g. function. This substitution of a single nucleotide is known as a missense variant.

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    This variant is a change at a specific point in the  gene from  to  resulting in incorrect NC_000002.12:g. function. This substitution of a single nucleotide is known as a missense variant.
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166277133C>T];[166277133C>T]" name="G2691A"> 
 
    # What does this mean?

    People with this variant have two copies of the [G2691A (Trp897Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6355/) variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    You are in the Severe Risk category. See below for more information.

    # How common is this genotype in the general population?

    <piechart percentage=? />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166277133=];[166277133=]" name="G2691A"> 
 
    # What does this mean?

    Your SCN9A gene has no variants. A normal gene is referred to as a "wild-type" gene.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=? />
  </Genotype>
<# G1376C #>
  <Genotype hgvs="NC_000002.12:g.[166286562G>C];[166286562=]" name="G1376C"> 

    # What does this mean?
 
    People with this variant have one copy of the [G1376C (Ser459Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6353/) variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=? />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166286562G>C];[166286562G>C]" name="G1376C"> 
 
    # What does this mean?

    People with this variant have two copies of the [G1376C (Ser459Ter)](https://www.ncbi.nlm.nih.gov/clinvar/variation/6353/) variant. This substitution of a single nucleotide is known as a missense mutation.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage=? />
  </Genotype>
  <Genotype hgvs="NC_000002.12:g.[166286562=];[166286562=]" name="G1376C"> 
 
    # What does this mean?

    Your G1376C gene has no variants. A normal gene is referred to as a "wild-type" gene.

    # What is the effect of this variant?

    You are in the Severe Risk category. See below for more information.

    # How common is this genotype in the general population?

    <piechart percentage=? />
  </Genotype>
<# unknown #>
  <Genotype hgvs="unknown"> 
 
    # What does this mean?

    Your SCN9A gene has an unknown variant.

    # What is the effect of this variant?

    The effect is unknown.

    # How common is this genotype in the general population?

    <piechart percentage= />
  </Genotype>
<# wildtype #>
  <Genotype hgvs="wildtype">
 
    # What does this mean?

    Your SCN9A gene has no variants. A normal gene is referred to as a "wild-type" gene.

    # What is the effect of this variant?

    Your variant is not associated with any loss of function.

    # How common is this genotype in the general population?

    <piechart percentage= />
  </Genotype>
</GeneAnalysis>

# How do changes in SCN9A affect people?

For the vast majority of people, the overall risk associated with the common SCN9A variants is small and does not impact treatment. It is possible that variants in this gene interact with other gene variants, which is the reason for our inclusion of this gene.

<# C984A (A;A) C829T (T;T) G2691A (A;A) G1376C (G;G) #>

| Variant       |Population %           | 
| :-------------: |:-------------:|
| G1376C (G;G) | Unknown |
| G2691A (A;A) | Unknown      |
| C829T (T;T) | 0.01%     |
| C984A (A;A)| 0.01%     |

# Severe Risk

[Congenital indifference to pain (CIP)](https://www.omim.org/entry/243000) is a rare disorder where individuals cannot feel pain, although they feel sensations of touch, hot, cold, and pressure.  They may have [frequent injuries](https://www.ncbi.nlm.nih.gov/pubmed/17167479) or [recurrent illness](https://www.ncbi.nlm.nih.gov/pubmed/22845492) and [ulcerations, which may result in the need for amputation](https://www.ncbi.nlm.nih.gov/medgen/C2752089) due to the inability to feel or respond appropriately to pain. 

# What should I do about this?

The [opioids](https://www.ncbi.nlm.nih.gov/pubmed/6462379) [naloxone](https://www.ncbi.nlm.nih.gov/pubmed/6085681) and [naltrexone](https://www.ncbi.nlm.nih.gov/pubmed/26634308) may allow patients to feel and respond to pain.  Other medicines used to SCN9A variants include [Lacosamide](https://www.drugbank.ca/drugs/DB06218), [Lidocaine](https://www.drugbank.ca/drugs/DB00281), [Ranolazine](https://www.drugbank.ca/drugs/DB00243), [Rufinamide](https://www.drugbank.ca/drugs/DB06201), [Valproic Acid](https://www.drugbank.ca/drugs/DB00313), and [Zonisamide]( https://www.drugbank.ca/drugs/DB00909). 

<#  C2986T (T;T) #>

| Variant      |Population %           | 
| :-------------: |:-------------:|
| C2986T (T;T) | 0.01% |

# Moderate Risk

This [variant](https://www.ncbi.nlm.nih.gov/clinvar/variation/6356/) causes three distinct diseases.  [Hereditary sensory and autonomic neuropathy type II (HSAN2)](https://www.ncbi.nlm.nih.gov/medgen/C2752089) causes progressively reduced response to pain, leading eventually to [frequent injuries](https://www.ncbi.nlm.nih.gov/pubmed/17167479), [recurrent illness](https://www.ncbi.nlm.nih.gov/pubmed/22845492), and [ulcerations, which require amputation](https://www.ncbi.nlm.nih.gov/medgen/C2752089).  [Generalized epilepsy with febrile seizures (type 7 plus)](https://www.ncbi.nlm.nih.gov/medgen/C2751777) causes severe seizures beginning between five months and four years of age.  [Paroxysmal extreme pain disorder (PEPD)](https://www.ncbi.nlm.nih.gov/pubmed/17145499) causes [rectal, eye, or jaw pain with flushing](https://www.ncbi.nlm.nih.gov/pubmed/1714549).  The pain attacks may last from seconds to hours and is considered a type of [peripheral neuropathy](https://www.ncbi.nlm.nih.gov/medgen/C1833661) as it affects the nervous system that connects the brain to sensory cells.

# What should I do about this?

Paroxysmal extreme pain disorder (PEPD) patients may consider trying [Carbamazepine](https://www.ncbi.nlm.nih.gov/pubmed/17145499).  They should also avoid [changes in temperature, emotional distress, spicy food, and cold drinks or food](https://www.ncbi.nlm.nih.gov/medgen/C1833661). Hereditary sensory and autonomic neuropathy type II (HSAN2) patients may consider the [opioids](https://www.ncbi.nlm.nih.gov/pubmed/6462379) [naloxone](https://www.ncbi.nlm.nih.gov/pubmed/6085681) and [naltrexone](https://www.ncbi.nlm.nih.gov/pubmed/26634308) that may cause increased sensitivity to pain.  

Other medicines used to SCN9A variants include [Lacosamide](https://www.drugbank.ca/drugs/DB06218), [Lidocaine](https://www.drugbank.ca/drugs/DB00281), [Ranolazine](https://www.drugbank.ca/drugs/DB00243), [Rufinamide](https://www.drugbank.ca/drugs/DB06201), [Valproic Acid](https://www.drugbank.ca/drugs/DB00313), and [Zonisamide]( https://www.drugbank.ca/drugs/DB00909).

<# T166298928G (G;G) #>

| Variant       |Population %           | 
| :-------------: |:-------------:|
| T166298928G (G;G)| 24.1%     |

# Moderate Risk

This variant causes perceived severe bodily pain, and may be linked with [small fiber neuropathy](https://www.ncbi.nlm.nih.gov/pubmed/29392201).  The [increased SLC9A membrane excitability, increased synaptic nerve efficacy, and reduced inhibition of neurons](https://www.ncbi.nlm.nih.gov/pubmed/22550986) is associated with [Central Sensitivity Syndromes (CSS)](https://www.ncbi.nlm.nih.gov/pubmed/22550986).  CSS is a [chronic, musculoskeletal condition](https://www.ncbi.nlm.nih.gov/pubmed/18191990/) that causes perceived increase in pain, hypersensitivity to painful stimuli, enhanced pain pathway function, [fatigue, insomnia, and mental distress](https://www.ncbi.nlm.nih.gov/pubmed/18191990).  Chronic diseases associated with CSS include [fibromyalgia](https://www.ncbi.nlm.nih.gov/pubmed/29392201), [irritable bowel syndrome (IBS), interstitial cystitis](https://www.ncbi.nlm.nih.gov/pubmed/24662556), and [ME/CFS](https://www.ncbi.nlm.nih.gov/pubmed/21951710/). 

# What should I do about this?

Chronic pain relief may include:
*  [Pre-emptive pain medications](https://www.ncbi.nlm.nih.gov/pubmed/22550986), including [acetaminophen](https://www.ncbi.nlm.nih.gov/pubmed/19410099/), opioids, and tramadol
*  [Nonsteroidal anti-inflammatory drugs](https://www.ncbi.nlm.nih.gov/pubmed/14997317/) 
*  [Tricyclic antidepressants](https://www.ncbi.nlm.nih.gov/pubmed/19410099/)
*  [Gabapentin, duloxetine or pregabalin](https://www.ncbi.nlm.nih.gov/pubmed/19410099/)
*  [Cyclobenzaprine, pregabalin, duloxetine, or milnacipran](https://www.ncbi.nlm.nih.gov/pubmed/19410099/)
*  [Multidisciplinary pain management programs](https://www.ncbi.nlm.nih.gov/pubmed/22550986), such as [cognitive behavioral therapy](https://www.ncbi.nlm.nih.gov/pubmed/11166973/)

<# symptoms fatigue D005221 pain D010146 muscle aches and pain D063806 joint pain without swelling or redness D018771 inflamation D007249 #>

<symptoms D005221 D010146 D063806 D018771 D007249 />

<# diseases inflammation	D007249 pain, congenital insensitivity or indifference to pain (CIP)	D000699 febrile seizures	D003294
 Disease susceptibility - increased susceptibility to viral, bacterial, and parasitical infections	D004198 epilepsy	D004827
pain, primary erythromelalgia (PERYTHM)	D004916 fibromyalgia	D005356 pain, Hereditary sensory and autonomic neuropathy type II (HSAN2)	pain, paroxysmal extreme pain disorder (PEPD D009477 pain  	D010146 pain, neuropathy	D010523 ME/CFS	D015673 interstitial cystitis	D018856 disease, Genetic Predisposition to Disease	D020022 irritable bowel syndrome (IBS)	D043183 pain, chronic pain	D059350 />

<diseases D007249 D000699 D003294 D004198 D004827 D004916 D005356 D009477 D010146 D010523 D015673 D018856 D020022 D043183 D059350 />
